package.json

```diff
@@ -21,10 +21,12 @@
   },
   "homepage": "https://github.com/northcoders/be-nc-news#readme",
   "devDependencies": {
+    "express": "^4.18.1",
```

Comment:
Express needs to be a full dependency and not a dev dependency. Our server relies on express to work so if this is pushed to production, we will lose this dependency and break the server. Just move this line further down to the 'dependencies' object in the package.json